tests/sql/moz-fx-data-shared-prod/telemetry_derived/urlbar_clients_daily_v1/test_overactive_filter/telemetry.clients_daily.yaml

```diff
@@ -12,9 +12,24 @@
   user_pref_browser_widget_in_navbar: true
   user_pref_browser_urlbar_suggest_searches: true
   user_pref_browser_urlbar_show_search_suggestions_first: true
-  scalar_parent_urlbar_picked_autofill_sum:
+  scalar_parent_urlbar_picked_autofill_about_sum:
```

Comment:
You'll have to add these to `tests/sql/moz-fx-data-shared-prod/telemetry_derived/urlbar_clients_daily_v1/telemetry.clients_daily.schema.json`, seems the format should be the same as `scalar_parent_urlbar_picked_autofill_sum`

Reply:
I'm not seeing it [here](https://github.com/mozilla/bigquery-etl/blob/urlbar-clients-daily/sql/moz-fx-data-shared-prod/telemetry_derived/urlbar_clients_daily_v1/schema.yaml#L58). Could you help out?

Reply:
See 875aba919233e4c175d1054a23e6118f19bc1cfa, does that look about right? Was `scalar_parent_urlbar_picked_autofill_sum` actually supposed to be replaced in this line?